Tests/VersioningTests/CommitTests.swift

```diff
@@ -9,13 +9,21 @@ final class CommitTests: XCTestCase {
         XCTAssertEqual(commit.message, "adding-optional-initialiser-for-icon")
     }
     
-    func testCommitTypes() throws {
+    func testCommitTypesWithoutDeps() throws {
         try CommitType.allCases.forEach {
             let message = "\($0.rawValue): commit message here"
             let commit = try Commit(string: message)
             XCTAssertEqual($0, commit.type)
         }
     }
+
+    func testCommitTypesWithDeps() throws {
```

Comment:
should this be `scope`, since that's what it's called in the conventional commit spec? https://www.conventionalcommits.org/en/v1.0.0/

> A scope may be provided to a commit’s type, to provide additional contextual information and is contained within parenthesis, e.g., feat(parser): add ability to parse arrays.